performance/reports/rp.py

```diff
@@ -42,14 +42,16 @@ def get_df_successes_by_rp(df_verifications_by_rp):
 
 
 def export_metrics_to_csv(df_export, report_output_path, date_start):
-    if not os.path.exists(report_output_path):
-        os.makedirs(report_output_path)
+    report_output_path_for_week = os.path.join(report_output_path, date_start)
```

Comment:
Can we have `output/rp_report/<date_start>` and lose the prefix/postfix from the file names maybe?

Reply:
I'll add a top-level folder for RP-Reports. The requirement for the filenames came from the PA's and seeing as they're the primary user for this I think that's probably not negotiable.